client/src/pages/Prospects.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from "react";
+import Navbar from "../components/Navbar";
+import { makeStyles } from "@material-ui/core/styles";
+import ProspectsTable  from '../components/prospects/Table';
+import {Button, Grid, IconButton, Snackbar, Typography, Container} from "@material-ui/core";
+
+export default function Prospects() {
+  const classes = useStyles();
+
+  return (
+      <div className={classes.root}>
+          <Navbar type="login">Don't have an account?</Navbar>
+          <Grid container spacing={3}>
+              <Grid item className="sidebar" sm={2}>
```

Comment:
I realized we shouldn't be using classNames with text like "sidebar", since we are using useStyles